spec/helpers/vimeo_helper_spec.rb

```diff
@@ -1,15 +1,5 @@
 require 'rails_helper'
 
-# Specs in this file have access to a helper object that includes
-# the VimeoHelper. For example:
-#
-# describe VimeoHelper do
-#   describe "string concat" do
-#     it "concats two strings with spaces" do
-#       expect(helper.concat_strings("this","that")).to eq("this that")
-#     end
-#   end
-# end
 RSpec.describe VimeoHelper, type: :helper do
-  pending "add some examples to (or delete) #{__FILE__}"
+  it "please to test grab_id method, @drvonnjerryxlii!"
```

Comment:
this is a note for future me, who will eventually experience boredom and go looking for specs to fill in.